cloud/azure/bin/setup-saml-keystore

```diff
@@ -38,6 +39,14 @@ if [[ ! "${resource_group}" ]] \
   exit 1
 fi
 
+if  key_vault::has_secret "${vault_name}" "${KEYSTORE_PASS_SECRET_NAME}" && 
+    storage::has_blob "${storage_account_name}" \
+    "${KEYSTORE_CONTAINER_NAME}" \
+    "${KEYSTORE_FILENAME}" ; then
```

Comment:
i think you need to remove the space here

Reply:
done

Reply:
ooo sorry i still see it!

Reply:
OK, it's def gone now!

Reply:
waitt is ittt?

Reply:
"${KEYSTORE_FILENAME}" ; then space after quote and before semi-colon